submissions/TaniaBondarenko/friends-app/app.js

```diff
@@ -0,0 +1,130 @@
+const numOfFriends=20;
+const apiUrl = `https://randomuser.me/api/?results=${numOfFriends}`;
+const friends = document.querySelector(".friends");
+let friendsData;
+let arrForFilter;
+let isFiltered = false;
+let justMale;
+let justFemale;
+
+function makeContainerEmpty() {
+        let innerText = " "; 
+        friends.innerHTML = innerText.replace(innerText, " ");    
+}
+
+function fetchFriend() {
+        fetch(apiUrl)
```

Comment:
This url can be inside this function because it is not used elsewhere in the app.

Reply:
Okay